src/mainwindow.cpp

```diff
@@ -2189,7 +2189,7 @@ void MainWindow::minimizeWindow()
 void MainWindow::QuitApplication()
 {
     m_noteEditorLogic->closeEditor();
-    MainWindow::close();
+    qApp->quit();
```

Comment:
Since this is a static method, I'd suggest:
```suggestion
    QCoreApplication::quit();
```